fluidreview/api_test.py

```diff
@@ -200,6 +201,59 @@ def test_process_user_both_exist_with_fluid_id(mocker):
     mock_save_profile.assert_not_called()
 
 
+@pytest.mark.parametrize(['amount_to_pay', 'award_cost', 'sends_email'], [
+    ['', '1000.10', False],
+    ['25.99', '', False],
+    ['', '', True]
```

Comment:
Should send an email as long as either `award_id` or `amount_to_pay` is a numerical value:
```python
    ['', '1000.10', True],
    ['25.99', '', True],
    ['', '', False]
```